contracts/MultiSigStub.sol

```diff
@@ -0,0 +1,67 @@
+pragma solidity ^0.4.15;
+
+/**
+ * @title MultiSigStub  
+ * Contact that delegates calls to a library to build a full MultiSigWallet that is cheap to create. 
```

Comment:
2. Thats MultiSigTokenWallet.sol
This address might be different for each network, but I can also deploy the contract in all networks at the same address. 
Notice that now Library contract is actually an working multisig that is initialized after the creation of the contract.